Docs/source/running_cpp/parameters.rst

```diff
@@ -280,7 +280,7 @@ Particle initialization
       symmetrize the beam in the x and y directions).
 
     * ``external_file``: inject macroparticles with properties (charge, mass, position, and momentum) according to data in external file.
-      It requires the additional argument ``<species_name>.injection_file``, which is the string corresponding to the OpenPMD file name.
+      It requires the additional arguments ``<species_name>.injection_file`` and ``<species_name>.q_tot``, which are the string corresponding to the openPMD file name and the beam charge.
```

Comment:
Suggested additional documentation of the current implementation for the user-docs:
```suggestion
      It requires the additional arguments ``<species_name>.injection_file`` and ``<species_name>.q_tot``, which are the string corresponding to the openPMD file name and the beam charge.
      The weighting of particles in the openPMD file will be ignored and re-assigned to match the beam charge.
```

Reply:
(can be done in a follow-up PR)